Source/WebKit/UIProcess/API/APINavigation.cpp

```diff
@@ -82,6 +85,7 @@ Navigation::Navigation(WebCore::ProcessIdentifier processID, Ref<WebBackForwardL
     , m_fromItem(WTFMove(fromItem))
     , m_backForwardFrameLoadType(backForwardFrameLoadType)
     , m_clientNavigationActivity(ProcessThrottler::TimedActivity::create(navigationActivityTimeout))
+    , m_requestStart(MonotonicTime::now())
```

Comment:
All these can be an initializer in the header instead of adding to each constructor.